src/gui.cpp

```diff
@@ -1417,27 +1420,47 @@ moby_model* gui::model_browser::render_selection_grid(
 
 void gui::model_browser::render_preview(
 		app& a,
-		GLuint* target,
+		GLuint* target,	
 		moby_model& model,
 		std::vector<texture>& textures,
 		const gl_renderer& renderer,
 		ImVec2 preview_size,
 		view_params params) {
-	glm::vec3 eye = glm::vec3(2.f * (1.1f - params.zoom), 0, 0);
+
+	auto draw_list = ImGui::GetWindowDrawList();
```

Comment:
Moving the fetching of draw_list here stops it being retrieved every iteration of the submodel loop. I don't know if it's performance heavy to do so, but it's a free win regardless.

Reply:
I've had a look at ImGui's implementation, and it doesn't seem to be doing anything very expensive. Also that code is only accessed for the model viewer anyway. I don't think it matters either way, but yea I think looking at the implementation for some of these functions is always a good idea.